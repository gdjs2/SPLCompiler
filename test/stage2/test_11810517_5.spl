int f(int x) {
	return 1;
}

int f(int x) {
	return x;
}

int main() {
	f(1);
	f(2);
}