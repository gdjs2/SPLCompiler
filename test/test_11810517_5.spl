float b;
float c

int main(int a){
    b=0;
    main();
}