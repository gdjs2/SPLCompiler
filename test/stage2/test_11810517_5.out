Error Type 4 at Line 5: function "f" is redefined.